com.google.EarthPro.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.google.EarthPro</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>Google Earth Pro</name>
+  <summary>3D planet viewer</summary>
+  <description>
+    <p>NOTE: This is an unofficial package based on the official binaries releases at www.google.com/earth. Create maps with advanced tools: Google Earth Pro on desktop is free for users with advanced feature needs. Import and export GIS data, and go back in time with historical imagery</p>
```

Comment:
usually the disclaimer is a separate paragraph at the end so that the description comes first

Reply:
updated